### PR TITLE
broot: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/broot.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install or reinstall the br shell function:
+
+`broot --install`
+
+- Navigate the directory:
+
+`br`
+
+- Show the size of files & directories:
```
Comment (kbdharun) [inline pages/linux/broot.md:14]:
```suggestion
- Display the size of files and directories:
```

File: pages/linux/broot.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`br -s`
+
+- Show permissions:
```
Comment (kbdharun) [inline pages/linux/broot.md:18]:
```suggestion
- Display permissions:
```
or maybe 
```suggestion
- Display permissions with owner and group:
```
Ref: [Arch wiki](https://man.archlinux.org/man/broot.1.en).

File: pages/linux/broot.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`br -p`
+
+- Only show directories:
```
Comment (kbdharun) [inline pages/linux/broot.md:22]:
```suggestion
- Display directories only:
```

File: pages/linux/broot.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`br -f`
+
+- Show hidden files & directories:
```
Comment (kbdharun) [inline pages/linux/broot.md:26]:
```suggestion
- Display hidden files and directories:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr @suyashbhawsar. Thanks for your contribution. I have some suggestions that need to be addressed.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks

**Comment by suyashbhawsar** [comment]
@kbdharun Thanks for the suggestions

File: pages/linux/broot.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# broot
+
+> A better way to navigate directories.
+> More information: <https://github.com/Canop/broot>.
+
+- Install or reinstall the br shell function:
```
Comment (CleanMachine1) [inline pages/linux/broot.md:6]:
```suggestion
- Install or reinstall the `br` shell function:
```

File: pages/linux/broot.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`broot --install`
+
+- Navigate the directory:
```
Comment (CleanMachine1) [inline pages/linux/broot.md:10]:
```suggestion
- Navigate the current directory:
```

File: pages/linux/broot.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`br`
+
+- Display the size of files and directories:
```
Comment (CleanMachine1) [inline pages/linux/broot.md:14]: Is this of all directories on the file system or just ones in the current working directory.

Comment (suyashbhawsar) [reply pages/linux/broot.md:14]: > Is this of all directories on the file system or just ones in the current working directory.

@CleanMachine1 It opens up at the current working directory but, you can navigate its child & even parent directory as it's a TUI program.

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**